### DISCUSSION TITLE
Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作

### DISCUSSION THREAD

**Discussion by lixiangyiIsNo1** [community/community#128505] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

IDE版本: WebStorm 2024.1.4
插件版本: 1.5.8.5775
内部版本号: #WS-241.17890.13
运行时版本: 17.0.11+1-b1207.24 amd64
VM: OpenJDK 64-Bit Server VM，JetBrains s.r.o.

我已经登录了Github账户, 我确认我有一个Copilot的授权, 但是从前几天开始, Copilot的上下文功能一直无法工作, 我没有收到任何错误提示

**Comment by lixiangyiIsNo1** [comment]
![image](https://github.com/community/community/assets/162766797/b79ff60a-b836-4fa9-b124-77d4a8ccf6eb)

**Comment by laoer** [comment]
一样的问题，JetBrains的几个软件里都不行。

**Comment by lixiangyiIsNo1** [comment]
我想要补充一个信息, copilot不是在所有类型的文件中都无法工作, 目前在vue和typescript文件中无法工作, 而在markdown文件中他是正常的

**Comment by laoer** [comment]
我是连对话都出不来，一直提示初始化中
<img width="604" alt="image" src="https://github.com/community/community/assets/1154962/37b973a6-a0d7-4dc7-87e8-bb03a777f203">

**Reply by lixiangyiIsNo1** [reply]
我们一样的

**Reply by laoer** [reply]
感觉就是今天出现的，升级了一下就这样了。

**Comment by lixiangyiIsNo1** [comment]
无法确定是插件还是软件的问题，我同样给webstorm提交了问题，他们让我找github



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月14日 18:22 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [community/community] Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作 (Discussion #128505) |

感觉就是今天出现的，升级了一下就这样了。

**Reply by laoer** [reply]
网络上有问题吗？

**Comment by lixiangyiIsNo1** [comment]
只在某些类型文件无法正常工作，不是网络的问题



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月14日 18:25 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [community/community] Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作 (Discussion #128505) |

网络上有问题吗？

**Reply by laoer** [reply]
我试了VS Code里的Copilot，是可以对话的，应该不是网络层的问题，估计还是插件或是IDE的问题，可能要等升级。

**Comment by lixiangyiIsNo1** [comment]
已经被修复了，webstrom已经推送了更新



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月14日 20:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [community/community] Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作 (Discussion #128505) |

我试了VS Code里的Copilot，是可以对话的，应该不是网络层的问题，估计还是插件或是IDE的问题，可能要等升级。

**Reply by laoer** [reply]
我没有收到推送更新，WebStrom没有，PyCharm也没有啊

**Reply by lixiangyiIsNo1** [reply, marked as answer]
我找到解决办法了
![image](https://github.com/community/community/assets/162766797/dab184a3-fc8b-43ea-9d58-cc4f0586d1ce)
修改这个值, 然后重启编辑器, 然后再修改回来, 再重启编辑器, 我就是这样解决的

**Reply by laoer** [reply]
试了，没用，我是Mac。

**Reply by lixiangyiIsNo1** [reply]
![image](https://github.com/community/community/assets/162766797/356eb695-de73-4155-8ce6-10ffe29eea94)
![image](https://github.com/community/community/assets/162766797/e48a74ce-5bdf-451b-b283-178c8aa4dbd9)
这是我的版本参数, 希望能帮到你

**Reply by laoer** [reply]
我和你的版本是一样的，只是我的是Mac版，只能等插件或是JetBrains更新了。

**Reply by laoer** [reply]
今天下午可以用了，不知道是什么问题。
中间重启过一次电脑。

**Comment by lixiangyiIsNo1** [comment]
我在家里的电脑是能正常使用的，公司的电脑一直出现这个问题



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月15日 08:53 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [community/community] Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作 (Discussion #128505) |

我没有收到推送更新，WebStrom没有，PyCharm也没有啊

**Comment by lixiangyiIsNo1** [comment]
你可以去看webstorm的更新记录，他们说已经修复了这个问题



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月15日 08:53 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [community/community] Github Copilot在WebStorm 2024.1.4中上下文生成功能, chat功能不能工作 (Discussion #128505) |

我没有收到推送更新，WebStrom没有，PyCharm也没有啊
